src/business.ts

```diff
@@ -0,0 +1,121 @@
+import type { Marqeta, MarqetaOptions, MarqetaError } from './'
+
+export interface BusinessIdentifications {
+  type: string;
+  value: string;
+}
+
+export interface IncorporationType {
+  incorporationType: string;
+  stateOfIncorporation: string;
+}
+
+export interface Address {
+  address1: string;
+  address2: string;
+  city: string;
+  state: string;
+  postalCode: string;
+  country: string;
+}
+
+export interface Personal {
+  firstName: string;
+  lastName: string;
+  home: Address;
```

Comment:
Is the home address required? If it's optional, then it would look like:

```typescript
 export interface Personal {
   firstName: string;
   lastName: string;
   home?: Address;
 }
```

and I think a lot of the things in these interfaces are optional... lots of them.

Reply:
Right, they're all optional, until KYB.

Reply:
it is optional, the review above is out-of-date